content/20.pharmaceuticals.md

```diff
@@ -657,7 +656,16 @@ Although these estimates are similar, it is important to note that they are draw
 A second systematic review of studies investigating TCZ treatment for COVID-19 analyzed 31 studies that had been published or released as pre-prints and reported that none carried a low risk of bias [@doi:10.1016/j.pulmoe.2020.07.003].
 Therefore, the present evidence is not likely to be sufficient for conclusions about the efficacy of TCZ.
 
-Additionally, there are possible risks associated with the administration of TCZ for COVID-19.
+On February 11, 2021, a preprint describing the first randomized control trial of TCZ was released as part of the RECOVERY trial [@doi:10.1101/2021.02.11.21249258].
+Of the 21,550 patients enrolled in the RECOVERY trial at the time, 4,116 adults hospitalized with COVID-19 across the 131 sites in the United Kingdom were assigned to the arm of the trial evaluating the effect of TCZ.
+Among them, 2,022 were randomized to receive TCZ and 2,094 were randomized to SOC, with 79% of patients in each group available for analysis at the time that the initial report was released.
+The primary outcome measured was 28-day mortality, and TCZ was found to reduce 28-day mortality from 33% of patients receiving SOC alone to 29% of those receiving TCZ, corresponding to a rate ratio of 0.86 (95% CI 0.77-0.96; _p_ = 0.007).
+TCZ was also significantly associated with the probability of hospital discharge within 28 days for living patients, which was 47% in the SOC group and 54% in the TCZ group ( (rate ratio 1.22, 95% CI 1.12-1.34, _p_ < 0.0001).
+A potential association between the TCZ administration in combination with a corticosteroid was observed, but the authors note that caution is advisable in light of the number of statistical tests conducted.
```

Comment:
This is a bit unclear - a potential association between TCZ administration in combination with a corticosteroid and _what_? It look like something is missing.

Reply:
```suggestion
A potential statistical interaction between TCZ and corticosteroids was observed, with the combination providing greater mortality benefits than TCZ alone, but the authors note that caution is advisable in light of the number of statistical tests conducted.
```